metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/sql/HiveConnectorFastPartitionService.java

```diff
@@ -112,7 +111,7 @@ public int getPartitionCount(
         final TableInfo tableInfo
     ) {
         if (context.getConfig().isIcebergEnabled() && HiveTableUtil.isIcebergTable(tableInfo)) {
-            throw new MetacatNotSupportedException("IcebergTable Unsupported Operation!");
```

Comment:
what the context of making this change?

Reply:
Bug was that this was throwing http 500 error instead of 415.